ResourceLocator/PeopleMap.html

```diff
@@ -0,0 +1,764 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Coveo Search</title>
+  <meta charset='utf-8'>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0,  maximum-scale=1.0, user-scalable=0">
+<!-- Needed libs for Coveo -->
+	<script src="lib/jsui/js/CoveoJsSearch.WithDependencies.js"></script> 
+	<script src="lib/jsui/js/templatesNew.js"></script> 
+	<link rel="stylesheet" href="lib/jsui/css/CoveoFullSearchNewDesign.css"/>
+<!-- End Needed libs for Coveo -->
+
+  <!-- Needed libs for Calendar/JQuery datepicker -->
+	<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
+	<script src="https://code.jquery.com/ui/1.11.3/jquery-ui.js"></script>
+	<link rel="stylesheet" href="/lib/calendar/fullcalendar.min.css">
+    <script src="lib/calendar/moment.min.js"></script>
+    <script src="lib/calendar/fullcalendar.min.js"></script>
+	<link rel="stylesheet" href="https://code.jquery.com/ui/1.11.3/themes/smoothness/jquery-ui.css"/>
+
+<!-- End Needed libs for Calendar/JQuery datepicker -->
+<!-- Needed libs for Google Maps -->
+	<script type="text/javascript"  src="https://maps.googleapis.com/maps/api/js?key=YOUROWNKEY"></script>
+	<script type="text/javascript" src="https://www.google.com/jsapi"></script>
+	<script src="lib/js/CoveoGoogleMap.js"></script> 
+	<script src="lib/js/CoveoMarkerclusterer.js"></script> 
+<!-- End Needed libs for Google Maps -->
+<!-- Connect to Search end point -->
+  <script type="text/javascript">
+    $(function () {
+     Coveo.SearchEndpoint.endpoints['default'] = new Coveo.SearchEndpoint({
+      restUri: 'https://platform.cloud.coveo.com/rest/search',
+      accessToken: 'YOURTOKEN',
+      queryStringArguments: {
+        organizationId: 'CLOUDORGID'
+      }
+    });
+      Coveo.$('#search').coveo('init');
+	
+    });
+  </script>
+</head>
+<body id="search" class="CoveoSearchInterface" data-enable-history="true" data-design="new">
+
+  <style>
```

Comment:
Can this be put in its own CSS file?

& Alignment is very off for many of those rules.